ACMILabs/interactive-label
# #10 Update Dockerfile.template for x86 from Spotlights config.

*Resolves issue #10*

Update Dockerfile.template for Intel Optiplex.

### Acceptance Criteria
- [x] Update `Dockerfile.template` for Intel Optiplex from Spotlights config

### Relevant design files
* None

### Testing instructions
1. Run `balena push s__interactive-label-x86`
1. See it builds okay and is running on: https://dashboard.balena-cloud.com/devices/92f18e471da44b4a0985a5c83b8478e4/summary

### DoD
For requester to complete:
